### DISCUSSION TITLE
请问大家dropdown的弹出面板如何可以和Select一样，自动调整位置使得完整显示

### DISCUSSION THREAD

**Discussion by taiyangzaitianshang** [ant-design/ant-design#42035] [Q&A]
以官网为例，默认向下弹出，如果下放空间不够的话，dropdown是截断的：
![image](https://user-images.githubusercontent.com/8625315/234751810-eee50bf5-d84a-4648-a5f3-86fef803ab07.png)

而Select的popup会自动向上弹出
![image](https://user-images.githubusercontent.com/8625315/234751729-558d696d-6232-412e-98ad-355fc563be2b.png)

请问有什么办法让dropdown也自动找位置弹出吗，看文档上有个autoAdjustOverflow属性，但是默认应该就是true了，没有效果

**Comment by zombieJ** [comment]
@MadCcc  在做了哈~

**Reply by taiyangzaitianshang** [reply]
> @MadCcc 在做了哈~

okk 辛苦了
